MemberExpectations.md

```diff
@@ -0,0 +1,58 @@
+It is important that members of our leadership groups (TSC, CommComm) act
+in a way that not only complies with the code of conduct but that supports
+the vision, mission and operating principles of the Node.js project.
```

Comment:
Do we have all these documented somewhere?

Reply:
It might also be helpful if those were links to the supporting documents, eg:

> in a way that not only complies with the [code of conduct](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md) but that supports the [vision](), [mission]() and [operating principles]() of the Node.js project.

Reply:
+1

Reply:
I'm up for putting together a first pass on a `values` document